node/external/blockAPI/baseBlock.go

```diff
@@ -162,11 +162,11 @@ func (bap *baseAPIBlockProcessor) getReceiptsFromMiniblock(miniblock *block.Mini
 	log.Debug(fmt.Sprintf("GetBulkFromEpoch took %s", time.Since(start)))
 
 	apiReceipts := make([]*transaction.ApiReceipt, 0)
-	for recHash, recBytes := range marshalizedReceipts {
-		receipt, errUnmarshal := bap.txUnmarshaller.UnmarshalReceipt(recBytes)
+	for _, pair := range marshalizedReceipts {
```

Comment:
suggestion to rename marshalizedReceipts to marshalledReceipts

Reply:
Renamed.